### ISSUE TITLE
Gundam Seed AniDB id 252 - ep mapping "3;8" instead of "3-8"

### ISSUE THREAD

**Issue by ZeroQI** [ScudLee/anime-lists#145]
```
  <anime anidbid="252" tvdbid="74474" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
    <name>Kidou Senshi Gundam SEED</name>
    <mapping-list>
      <mapping anidbseason="0" tvdbseason="0">;1-4;2-7;3;8</mapping>
    </mapping-list>
    <supplemental-info>
      <studio>Sunrise</studio>
    </supplemental-info>
  </anime>
```

**Comment by ZeroQI** [comment]
Should be ";1-4;2-7;3-8;"

**Comment by ScudLee** [comment]
Should be fixed in f4f26bf60100fdc4638cfb7dcebe3466795b0ff0

Thanks!
